src/commands/protocols/get/sender/local-get-command.js

```diff
@@ -43,6 +43,14 @@ class LocalGetCommand extends Command {
                     OPERATION_ID_STATUS.COMPLETED,
                 );
 
+                /* JUST FOR TESTING
```

Comment:
Should we uncomment this?

Reply:
Not sure when this error needs to be thrown. Should i wrap lines 25-30 in try-catch and execute the commented out code if there's an error?

Reply:
Ok, you can just remove this commented line than, i thought that it was commented some previous code